metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -1168,9 +1171,25 @@ class MetalsLspService(
         None
     }
 
-    uriOpt match {
-      case Some(uri) => {
-        val path = uri.toAbsolutePath
+    val pathOpt = uriOpt.flatMap { uri =>
+      try {
+        Some(uri.toAbsolutePath)
```

Comment:
This makes me think, maybe we should use `toAbsolutePathSafe` which would take in workspace and create the report automatically? `toAbsolutePath` is used in a lot of places :thinking: Or do you think since this method is heavily used we should be safe to jsut report it here?